src/components/tipsslider/tipsslider.jsx

```diff
@@ -0,0 +1,64 @@
+var classNames = require('classnames');
+var defaults = require('lodash.defaults');
+var React = require('react');
+var Slider = require('react-slick');
+
+var Thumbnail = require('../thumbnail/thumbnail.jsx');
+
+require('slick-carousel/slick/slick.scss');
+require('slick-carousel/slick/slick-theme.scss');
+require('./tipsslider.scss');
+
+var TipsSlider = React.createClass({
```

Comment:
small nitpick – it could be good to call it something like `NestedCarousel`, or in some way that describes a bit how it differs from the other Carousel component.